wrapanapi/systems/nuage.py

```diff
@@ -24,75 +19,61 @@ class NuageSystem(System):
     """
 
     _stats_available = {
-        'num_security_group': lambda self: len(self.list_security_groups()),
-        'num_cloud_subnet': lambda self: len(self.list_cloud_subnets()),
-        'num_cloud_tenant': lambda self: len(self.list_enterprises()),
-        'num_network_router': lambda self: len(self.list_domains())
+        'num_security_group': lambda self: self.api.policy_groups.count()[2],
```

Comment:
Please include in-code comment about why you're returning the `[2]` index from the `count()` call, for future reference in case of API changes.